src/components/CreateVacancyModal/CreateVacancyModal.tsx

```diff
@@ -7,56 +7,69 @@ import {
     Form,
     FormAction,
     FormActions,
-    FormInput,
     FormTitle,
+    Input,
     Modal,
     ModalContent,
     ModalCross,
     ModalHeader,
     Text,
+    nullable,
 } from '@taskany/bricks';
-import { gapM, gapS, gapXs, gray3, gray8 } from '@taskany/colors';
+import { Group, VacancyStatus } from 'prisma/prisma-client';
+import { danger0, gapM, gapS, gray3, gray8 } from '@taskany/colors';
 
-import { useRouter } from '../../hooks/useRouter';
-import { GroupComboBox } from '../GroupComboBox/GroupComboBox';
 import { useVacancyMutations } from '../../modules/vacancyHooks';
 import { CreateVacancy, createVacancySchema } from '../../modules/vacancySchemas';
 import { HireStreamComboBox } from '../HireStreamComboBox/HireStreamComboBox';
+import { UserComboBox } from '../UserComboBox/UserComboBox';
+import { RoleComboBox } from '../RoleComboBox/RoleComboBox';
+import { VacancyStatusComboBox, statusesMap } from '../VacancyStatusComboBox/VacancyStatusComboBox';
+import { GroupSupervisor } from '../../modules/groupTypes';
 
 import { tr } from './CreateVacancyModal.i18n';
 
 interface CreateVacancyModalProps {
     visible: boolean;
     onClose: VoidFunction;
+    group: Group & GroupSupervisor;
 }
 
 const NoWrap = styled.div`
     white-space: nowrap;
 `;
 
 const StyledInputContainer = styled.div`
-    display: flex;
+    display: grid;
+    grid-template-columns: 120px 1fr 1fr;
     gap: ${gapS};
+    width: 100%;
     align-items: center;
-    padding: ${gapXs} ${gapM};
+    background-color: var(--gray3);
+    padding-left: ${gapM};
+    padding-bottom: ${gapS};
     background-color: ${gray3};
 `;
 
-export const CreateVacancyModal = ({ visible, onClose }: CreateVacancyModalProps) => {
+export const CreateVacancyModal = ({ visible, onClose, group }: CreateVacancyModalProps) => {
     const { createVacancy } = useVacancyMutations();
-    const router = useRouter();
 
     const {
-        register,
         handleSubmit,
         reset,
         setValue,
         formState: { errors, isSubmitting, isSubmitSuccessful },
-    } = useForm<CreateVacancy>({ resolver: zodResolver(createVacancySchema) });
+    } = useForm<CreateVacancy>({
+        defaultValues: {
+            groupId: group.id,
+            hiringManagerId: group.supervisorId || undefined,
```

Comment:
you use two fields out of the whole object - it's probably better to pass only them.

Reply:
UserComboBox uses group supervisor to set default value of hiringManager and groupName used in header of modal